app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -156,6 +156,7 @@
     <icon drawable="@drawable/calculator" package="com.oneplus.calculator" name="Calculator" />
     <icon drawable="@drawable/calculator" package="com.sec.android.app.popupcalculator" name="Calculator" />
     <icon drawable="@drawable/calculator_plus" package="com.gigaworks.tech.calculator" name="Calculator Plus" />
+    <icon drawable="@drawable/calendar" package="foundation.e.calendar" name="Calendar" />
```

Comment:
Calendar icons had been handled by #750, you should submit changes to Lawnchair like https://github.com/LawnchairLauncher/lawnchair/pull/3038.

Reply:
Alright, I'm removing the calendar change now, you can send a new PR to Lawnchair.